Sources/ArcGISToolkit/Components/Offline/PreplannedMapModel.swift

```diff
@@ -43,6 +52,11 @@ class PreplannedMapModel: ObservableObject, Identifiable {
     /// A Boolean value indicating if a user notification should be shown when a job completes.
     let showsUserNotificationOnCompletion: Bool
     
+    /// The file size of the preplanned map area.
+    var directorySize: Int? {
+        FileManager.default.sizeOfDirectory(at: mmpkDirectory)
```

Comment:
I didn't subtract the metadata.json and thumbnail.png size as they take space on disk as well.